src/components/TextInputComponent/TextInputComponent.tsx

```diff
@@ -1,12 +1,14 @@
 import { forwardRef, type InputHTMLAttributes, type KeyboardEventHandler, useMemo } from 'react';
 import './TextInputComponent.css';
-
 interface Props extends InputHTMLAttributes<HTMLInputElement> {
-	label: string;
+	label?: string;
```

Comment:
fix: As per the WCGA standard, labels are compulsory for inputs like text, email, password, etc.
```suggestion
	label: string;
```